terracumber_config/tf_files/SUSEManager-Test-Orion-Reporting-Hub.tf

```diff
@@ -0,0 +1,211 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-Orion-acceptance-tests-reportdb-hub"
+}
+
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='Uyuni' && run-testsuite #TODO run only sanity, core and hub tests, including reporting"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/uyuni-project/uyuni.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "hub-reporting-beta3"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results TEST-HUB $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results TEST-HUB : Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://salzbreze.mgr.suse.de/system"
+}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  images = ["centos7o", "opensuse152o", "opensuse153o","sles15sp3o", "sles15sp4o"]
+  use_avahi    = false
+  name_prefix  = "suma-reportdb-"
+  domain       = "mgr.suse.de"
+
+  provider_settings = {
+    pool         = "ssd"
+    network_name = null
+    bridge       = "br0"
+  }
+}
+
+module "hub-srv" {
+  source = "./modules/server"
+  base_configuration = module.base_core.configuration
+  name = "hub-srv"
+  product_version = "head"
+  additional_repos = {
+    Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/TEST:/Orion/SLE_15_SP4/"
+  }
+  image = "sles15sp4o"
+  provider_settings = {
+    mac = "aa:b2:93:01:01:42"
+  }
+  server_configuration = {
+    hostname = "suma-reportdb-hub-srv.mgr.suse.de"
+  }
+}
+
+module "peripheral1" {
+  source = "./modules/server"
+  base_configuration = module.base_core.configuration
+  product_version = "head"
+  name = "peripheral1"
```

Comment:
Can we use an easier name? I cannot pronounce it nor remember how to write it.

"per1" or another shortcut please.

Reply:
or "prh1"